London |May-2025|Merhawit Teages | Wireframe

merhawit-teages opened the pull request:
<!--

You must title your PR like this:

REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME

For example,

London | May-2025 | Carol Owen | Wireframe

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

✅  I have committed my files one by one, on purpose, and for a reason
✅I have titled my PR with REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME
✅ I have tested my changes
✅ My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)
✅ My changes meet the [requirements](./README.md) of this task

## Changelist

Updated the modal layout and styling to make the modal container smaller. Adjusted the width and margin to make the modal more centered and responsive. Added proper close button functionality with styling. Additionally, added links for the images and updated the "Learn more" buttons with correct external links for further details.


## Questions
Please let me know if there are any further adjustments needed.

illicitonion commented on Wireframe/index.html:
```diff
@@ -8,26 +8,74 @@
+          target="_blank"
+          >Learn more about README files</a
+        >
+        <!-- External Link -->
```
What benefit is this comment providing?

illicitonion reviewed (commented):
This is generally looking good, but a few things to look at:
1. The page doesn't currently score 100% for lighthouse and it's meant to.
2. The wireframe describes show one article at the top, and then two thinner ones underneath side-by-side, but yours shows three of the same size in a different arrangement.
3. Your footer background seems to start offset from the left of the screen, and accordingly your footer text isn't centred.

Please fix these things up and ask for review again.

illicitonion reviewed (commented):
This is generally looking really good.

One visual issue is that the requirements say the footer is meant to be fixed at the bottom of the screen, but yours isn't - I need to scroll down to see it.

Also, I ran lighthouse on your page for accessibility and it only scored 93% can you take a look?

Thanks!

merhawit-teages commented:
I’ve removed all width and height attributes from the HTML and moved the styling to CSS as suggested.Also cleaned up the indentation across the stylesheet for consistency.

I’ve updated the footer to ensure it stays fixed at the bottom of the screen, and ran Lighthouse again

illicitonion commented on Wireframe/index.html:
```diff
@@ -3,31 +3,80 @@
+      <!-- Top article (full width) -->
+      <article class="top-article">
+        <img
+          src="/Users/mera/Downloads/tinified (2)/Readme .png"
```
These images don't load because they're not included in your repo, and are trying to look on the viewer's computer. Please add the images to this repo, and update the `src` attribute so they load from here.

illicitonion reviewed (commented):
This is looking good - one thing to fix up around the images.

Also, your branch now includes the Form-Controls exercises - please undo this (maybe using `git rebase -i`) so that each PR you make only contains one project.

merhawit-teages commented:
I’ve added the missing images directly into the repository and updated the image paths in the HTML so they load correctly.
I’ve also removed the Form-Controls content from this branch, so it only contains the Wireframe project as requested.
